controlplane/src/core/repositories/analytics/TraceRepository.ts

```diff
@@ -78,4 +78,45 @@ export class TraceRepository {
       },
     }));
   }
+
+  public async getRouterVersion(organizationID: string, startDate: string): Promise<string> {
+    const parsedDateRange = isoDateRangeToTimestamps({ start: startDate, end: startDate });
+    const [start] = getDateRange(parsedDateRange);
+
+    const queryWithTimeConstraint = `
+    WITH 
+      toDateTime('${start}') AS startDate
+    SELECT  
+        SpanAttributes['wg.router.version'] as routerVersion
+    FROM ${this.client.database}.otel_traces
```

Comment:
Using `otel_traces` is not correct because the retention of this table is much smaller than the materialized view.

Reply:
I'd also prefer to use the request metrics MV because it is not subject to sampling and has less data to scan.

Reply:
we dont store it in request metrics mv